src/templates/base/2019/contributors.html

```diff
@@ -262,27 +271,37 @@ <h1 class="title title-lg">
   </div>
   <div class="contributors {{ ' '.join(config.teams.keys()) }}">
     <div class="filter-container">
-      <p>
+      <p aria-live="polite" aria-atomic="true">
```

Comment:
Clever. I like it.

Reply:
All @nico3333fr’s idea so full credit to him!